components/brave_wallet/browser/simulation_responses.idl

```diff
@@ -16,19 +16,26 @@ namespace simulation_responses {
   enum WarningKind {
     APPROVAL_TO_E_O_A,
     BLOCKLISTED_DOMAIN_CROSS_ORIGIN,
+    BLUR_BULK_ORDER_NOT_ON_BLUR,
+    BLUR_V2_ORDER_NOT_ON_BLUR,
     BULK_APPROVALS_REQUEST,
     COMPROMISED_AUTHORITY_UPGRADE,
     COPY_CAT_DOMAIN,
     COPY_CAT_IMAGE_UNRESPONSIVE_DOMAIN,
     DANGLING_APPROVAL,
-    DEVTOOLS_DISABLED,
+    DEBUGGER_PAUSED,
+    DURABLE_NONCE,
+    DEVTOOLS_DISABLED, // TODO(onyb): DEPRECATED
     ETH_SIGN_TX_HASH,
+    FORTA,
+    IMBALANCED_DOLLAR_VALUE,
     KNOWN_MALICIOUS,
-    MAINNET_REPLAY_POSSIBLE,
+    MALICIOUS_PACKAGES,
+    MAINNET_REPLAY_POSSIBLE, // TODO(onyb): DEPRECATED
     MULTI_COPY_CAT_DOMAIN,
     NEW_DOMAIN,
-    NON_ASCII_URL,
-    OBFUSCATED_CODE,
+    NON_ASCII_URL, // TODO(onyb): DEPRECATED
+    OBFUSCATED_CODE, // TODO(onyb): DEPRECATED
     PERMIT_NO_EXPIRATION,
```

Comment:
why we keep deprecated ones here?

Reply:
Thanks for pointing out. Removed them here: https://github.com/brave/brave-core/pull/21609/commits/1e230b692bde2fac459f5550e281ba3d2768a4c6